test/e2e/page-objects/pages/homepage.ts

```diff
@@ -283,13 +283,15 @@ class HomePage {
     );
   }
 
-  async check_ganacheBalanceIsDisplayed(
-    ganacheServer?: Ganache,
+  async check_localBlockchainBalanceIsDisplayed(
+    localBlockchainServer?: Ganache,
```

Comment:
this param will eventually accept both Ganache or Anvil classes. See [here](https://github.com/MetaMask/metamask-extension/pull/27246/files#diff-907f87ff9f4852f2c65855968d6f7a5028f728eaeb370239cdabc6a57c139342R288)